brambling/models.py

```diff
@@ -775,17 +784,44 @@ class Payment(models.Model):
     card = models.ForeignKey('CreditCard', blank=True, null=True)
     is_confirmed = models.BooleanField(default=False)
 
+    def refund(self, amount, issuer, dwolla_pin=None):
```

Comment:
The refund is always listed as the same method as the payment. Which is a little weird and not necessarily accurate. For example, I might accept payment in cash but refund someone via paypal (hypothetically).